spec/unit/generator/xcconfig/aggregate_xcconfig_spec.rb

```diff
@@ -157,8 +157,8 @@ def spec
               fixture_spec('monkey/monkey.podspec')
             end
 
-            it 'does not add the framework build path to the xcconfig' do
-              @xcconfig.to_hash['FRAMEWORK_SEARCH_PATHS'].should.be.nil
+            it 'does add the framework build path to the xcconfig' do
+              @xcconfig.to_hash['FRAMEWORK_SEARCH_PATHS'].should.not.be.nil
             end
```

Comment:
`monkey.podspec` has been updated to reference a dynamic `vendored_framework`. As such, `FRAMEWORK_SEARCH_PATHS` should no longer be nil.